### PR TITLE
Displayed properties of node on editor

### PR DESCRIPTION
**Pull request by nehashende**

### CODE REVIEW

File: src/browser/modules/EditorInfo/DisplayNodeDetails.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  Drawer,
+  DrawerHeader,
+  DrawerSection,
+  DrawerSubHeader,
+  DrawerBody
+} from 'browser-components/drawer/index'
+import { getStringValue } from './utils'
+import * as _ from 'lodash'
+
+function DisplayNodeDetails (props) {
```
Comment (rahul-tarafdar-synerzip) [inline src/browser/modules/EditorInfo/DisplayNodeDetails.jsx:18]: Add JSDoc for this method

File: src/browser/modules/EditorInfo/EditorInfo.jsx [hunk 2]
```diff
@@ -5,11 +5,24 @@
 import React, { Component } from 'react'
 import { connect } from 'react-redux'
 import { withBus } from 'react-suber'
+import DisplayNodeDetails from './DisplayNodeDetails'
 
 export class EditorInfo extends Component {
```
Comment (rahul-tarafdar-synerzip) [inline src/browser/modules/EditorInfo/EditorInfo.jsx:12]: Add JSDoc for this method

File: src/browser/modules/EditorInfo/DisplayNodeDetails.jsx (hunk 1 above)
Comment (nehashende) [reply src/browser/modules/EditorInfo/DisplayNodeDetails.jsx:18]: Done with adding the required JSDoc.

File: src/browser/modules/EditorInfo/EditorInfo.jsx (hunk 2 above)
Comment (nehashende) [reply src/browser/modules/EditorInfo/EditorInfo.jsx:12]: Done with adding the required JSDoc.

**Review by rahul-tarafdar-synerzip — APPROVED** [review]

**Comment by rahul-tarafdar-synerzip** [comment]
resolve the conflict

**Review by rahul-tarafdar-synerzip — APPROVED** [review]

**Pull request merged**